EIPS/eip-2565.md

```diff
@@ -15,22 +15,62 @@ requires: 198
-3. Improving the underlying libraries beneath the ModExp Precompile
-4. Any combination of (1), (2), and (3)
-
-We recommend **Option (1) and (2)** which provides a large practical improvement to gas estimation while keeping implementation complexity low. Option (3) can be implemented at a later point to improve the gas pricing even further.
+* Changing the value of the `GQUADDIVISOR` parameter in the ModExp pricing formula to bring its costs more in-line with other precompiles
 
 ## Motivation
 Modular exponentiation is a foundational arithmetic operation for many cryptographic functions including signatures, VDFs, SNARKs, accumulators, and more. Unfortunately, the ModExp precompile is currently over-priced, making these operations inefficient and expensive. By reducing the cost of this precompile, these cryptographic functions become more practical, enabling improved security, stronger randomness (VDFs), and more.
 
 ## Specification
-The current gas pricing formula is defined in EIP-198. This formula divides a ‘computational complexity’ function by a ‘gas conversion’ parameter called `GQUADDIVISOR` to arrive at a gas cost: `floor(mult_complexity(x)/GQUADDIVISOR)`
 
-### **Recommended** Option (1): Modify ‘computational complexity’ function and add minimum gas cost
+The current gas pricing formula is defined in [EIP-198](eip-198.md) as: 
+
+```
+floor(mult_complexity(max(length_of_MODULUS, length_of_BASE)) * max(ADJUSTED_EXPONENT_LENGTH, 1) / GQUADDIVISOR)
+```
+
+At `fork_block`, 
+ - set `GQUADDIVISOR` to `200`.
+
+
+## Rationale
+
+With this change, the cost of the ModExp precompile will have a higher cost (gas/second) than other precompiles such as ECRecover.
+
+![Option 2 Graph](../assets/eip-2565/GQuad_Change.png)
+
+## Test Cases
+
+There are no changes to the underlying interface or arithmetic algorithms, so the existing test vectors can be reused. Gas values will need to be updated in the existing unit tests based on the final pricing decided in this EIP. This will ensure that the updated gas calculations are done correctly.
+
+Existing testvectors from [go-ethereum](https://github.com/ethereum/go-ethereum/blob/master/core/vm/testdata/precompiles/modexp.json):
+
+| testcase | EIP-198 gas| 2565 opt  1 gas| 2565 opt 2 gas | 
+| -|--|--|--
+|modexp_nagydani_1_square| 204 | 100| 100?|
+
+TODO fill this table
+
+
+
+## Security Considerations
+
+The biggest security consideration for this EIP is  creating a potential DoS vector by making ModExp operations too inexpensive relative to their computation time.
+
+To address this, clients should perform benchmarking to ensure that gas limits are not causing DoS problems. 
+
+## References
+[EIP-198](https://eips.ethereum.org/EIPS/eip-198) 
+
+## Copyright
+Copyright and related rights waived via [CC0](https://creativecommons.org/publicdomain/zero/1.0/).
+
+## Appendix
+
+### Abandoned option : Modify ‘computational complexity’ function and add minimum gas cost
+
```

Comment:
Consider moving this to the rationale (that section is intended to represent why certain choices were made in this EIP) rather than a non-standard Appendix section.